### ISSUE TITLE
works partially on OSX/BSD

### ISSUE THREAD

**Issue by Morantron** [Morantron/tmux-fingers#7]
Seems like `0.1.0`is not working at all on OSX. I need to investigate the issue and setup automated tests to catch this problems.

EDIT: `0.1.1` partially fixes the issues, I still need to find out why some commands like `ifconfig` yield a blank screen.

**Comment by chrisjunlee** [comment]
What's your current and planned automated test setup?

**Comment by Morantron** [comment]
>What's your current and planned automated test setup?

There is no automated test setup at this moment.

Next steps would be to setup some automated integration tests ( probably [expect tool](http://expect.sourceforge.net/) from what I've seen in other tmux plugins ) and setup travis for continuous integration.

I still need to figure out what the differences are. AFAIK I need to make everything fully POSIX compliant. So I'll need to setup some VM/containers to ensure everything works in both environments.

This would be very important since I plan in the future to rewrite a core part of the script to improve the performance, which is very poor right now. It'd be awesome to not breaking everything after that refactor :trollface: 

Any help or ideas would be appreciated!

**Comment by jesus-** [comment]
Well just install in OSX and it basically kills all my session opened... Too bad, the idea is quite good. (just commented to receive updates)

**Comment by chrisjunlee** [comment]
@jesus- : That's odd, there's nothing in the source that would kill tmux sessions. 

What's your config and repro steps?

**Comment by Morantron** [comment]
@jesus- Ensure you have at least tmux 2.1. I tried once in OSX with 1.9 and everything went :boom:, sessions killed and everything.

**Comment by jesus-** [comment]
yes... I have 1.9a. Thanks!

**Comment by jarandaf** [comment]
First of all, kudos @Morantron, awesome work. Just wanted to corroborate that commands like `ifconfig` on OSX yield the well known blank screen.

**Comment by Morantron** [comment]
I've narrowed the culprit to `grep -n`.

For this test text

```bash
# test-file.txt
2016 201234 1234
test test test

wtf


wwowowo wo wtf  wo wtf
```

In GNU grep:

```
➜ ~/hacking/tmux-fingers cat test-grep.txt | grep -oniE [0-9]{4}
1:2016
1:2012
1:1234
```

In BSD:

```
[vagrant@ ~]$ cat test-grep.txt | grep -oniE --line-number [0-9]{4}
1:2016
2012
1234
```

Looks like POSIX grep does not number all matches. I wonder if I can make POSIX grep to output linenumber to all matches. Otherwise I'll need to format the output of grep with AWK.

**Comment by Morantron** [comment]
Should be fixed now on 0.1.2 version! :tada:
